### ISSUE TITLE
Add on screen label to configurable controls

### ISSUE THREAD

**Issue by aernan** [sharebrained/portapack-hackrf#45]
The top bar of things you can set seem like frequency , gain ???, volume? 
Please display the label for whatever value is currently being adjusted. Could you put the text right justified in the blue bar area?

**Comment by jboone** [comment]
Good idea. I've started making space in that area for more informative labeling.

**Comment by KC5CQW** [comment]
Is there any way to label the bar graphs? I understand this may require resizing or relocation. Perhaps they my be independently selected for on screen view or omission via menu check box?...
